framework/src/utils/RankTwoTensor.C

```diff
@@ -480,45 +480,49 @@ RankTwoTensor::positiveProjectionEigenDecomposition(std::vector<Real> & eigval,
   // The calculate of projection tensor follows
   // C. Miehe and M. Lambrecht, Commun. Numer. Meth. Engng 2001; 17:337~353
 
-  // Compute eigenvectors and eigenvalues of this tensor
-  (*this).symmetricEigenvaluesEigenvectors(eigval, eigvec);
+  // Compute eigenvectors and eigenvalues of mechanical strain
```

Comment:
Don't make this change. This is a generic rank two tensor routine, it is _not_ specific to "mechanical strain"

Reply:
It was me who changed it to "this tensor", but I somehow copied the older code... Thanks for the catch!